modules/customizing-the-web-console-URL.adoc

```diff
@@ -97,3 +97,7 @@ spec:
 <2> Reference to a secret in the `openshift-config` namespace that contains a TLS certificate (`tls.crt`) and key (`tls.key`). This is required if the domain for the custom hostname suffix does not match the cluster domain suffix. The secret is optional if the suffix matches.
 
 . Save the file to apply the changes.
+
+== Recognizing resource/project limits and quotas
+
+A message appears in the `Topology` or `List` views when a deployment reaches its preset limits or quotas. 
```

Comment:
Depending on the context, Topoology and List might need to be in asterisks (*) and not backticks. I couldn't find any instances of `List` view in the docs, and I think they might be GUI elements anyways?

Reply:
Fixed.
![image](https://user-images.githubusercontent.com/83772007/231260289-3c83c466-d056-4c07-9c63-ad436849933e.png)

Reply:
I second this:
(and double check that "List" is actually accurate, or what that is supposed to be)

I'm not sure where that is. I'd also maybe specifically call out that *Topology* is in the *Developer* perspective of the web console, and that *List* is wherever it is (not sure where that's referring to).

Reply:
Fixed. Added more description to how the message is indicated and where to find List view.